src/stage0.txt

```diff
@@ -12,7 +12,7 @@
 # source tarball for a stable release you'll likely see `1.x.0` for rustc and
 # `0.x.0` for Cargo where they were released on `date`.
 
-date: 2018-01-02
+date: 2018-01-18
```

Comment:
Shouldn't that be 2018-02-18 rather that 01?

Reply:
Yes, thank you.